iOS/Objective C/IronSourceDemoApp/DemoViewController.m

```diff
@@ -7,27 +7,32 @@
 //
 
 #import "DemoViewController.h"
-#import "DemoInitializationDelegate.h"
 #import "DemoRewardedVideoAdDelegate.h"
 #import "DemoInterstitialAdDelegate.h"
 #import "DemoBannerAdDelegate.h"
 #import "DemoImpressionDataDelegate.h"
 
-#define kDefaultUserId @"demoapp"
+// Replace with your app key as available in the LevelPlay dashboard
 #define kAppKey @"8545d445"
 
+// Replace with your ad unit ids as available in the LevelPlay dashboard
+#define kInterstitialAdUnitId @"wmgt0712uuux8ju4"
+#define kBannerAdUnitId @"iep3rxsyp9na3rw8"
+
 @interface DemoViewController ()
 
 @property (nonatomic, strong) DemoRewardedVideoAdDelegate     *rewardedVideoDelegate;
-@property (nonatomic, strong) DemoInterstitialAdDelegate      *interstitialDelegate;
-@property (nonatomic, strong) DemoBannerAdDelegate            *bannerDelegate;
-
-@property (nonatomic, strong) DemoInitializationDelegate    *initializationDelegate;
+@property (nonatomic, strong) DemoInterstitialAdDelegate      *interstitialAdDelegate;
+@property (nonatomic, strong) DemoBannerAdDelegate            *bannerAdViewDelegate;
 
 @property (nonatomic, strong) DemoImpressionDataDelegate    *impressionDataDelegate;
 
 @property (nonatomic, strong) ISPlacementInfo           *rewardedVideoPlacementInfo;
-@property (nonatomic, strong) ISBannerView              *bannerView;
+
+@property (nonatomic, strong) LPMBannerAdView             *bannerAdView;
+@property (nonatomic, strong) LPMAdSize                   *bannerSize;
+@property (nonatomic, strong) LPMInterstitialAd           *interstitialAd;
```

Comment:
The properties are divided to sections so I would combine the banner properties and interstitial properties on different sections.